src/result_collection.jl

```diff
@@ -43,9 +43,15 @@ See also [`collect_results`](@ref).
 * `special_list = []`: List of additional (derived) key-value pairs
   to put in `df` as explained below.
 
-`special_list` is a `Vector{Pair{Symbol, Function}}` where each entry
-is a derived quantity to be included in `df`. The function entry always
+`special_list` is a `Vector` where each entry
+is a derived quantity to be included in `df`. There are two types of entries.
+The first option is of the form `key => func` where the `key` is a symbol
+to be used as column name in the DataFrame. The function entry always
 takes a single argument, which is the loaded result-file (a dictionary).
+The second option is to provide just one function `func`. This function
+also takes the single dictionary argument but returns one or more
+`key => value` pairs. This second notation may be useful when a single
+when one wants to extract values for multiple columns in a single step.
```

Comment:
The last sentence here doesn't make sense (grammatically).

Reply:
fixed it